src/wp-includes/widgets/class-wp-widget-custom-html.php

```diff
@@ -202,44 +196,22 @@ public function update( $new_instance, $old_instance ) {
 	/**
 	 * Loads the required scripts and styles for the widget control.
 	 *
-	 * @since 4.9.0
+	 * @since CP-2.3.0
 	 */
 	public function enqueue_admin_scripts() {
-		$settings = wp_enqueue_code_editor(
-			array(
-				'type'       => 'text/html',
-				'codemirror' => array(
-					'indentUnit' => 2,
-					'tabSize'    => 2,
-				),
-			)
-		);
+		if ( ! isset( $this->number ) ) {
+			return;
+		}
 
+		wp_enqueue_code_editor( array( 'type' => 'text/html' ) );
+		wp_enqueue_script( 'wp-codemirror' );
```

Comment:
I think this should be a condition enqueue based on user level setting, I can push a change.